person.ru

```diff
@@ -0,0 +1,20 @@
+class Person
```

Comment:
- [ ] It would be better if we change the file extensions to our ruby files to `.rb` instead of `.ru` as we have now